src/routes/docs/advanced/platform/release-policy/+page.markdoc

```diff
@@ -0,0 +1,161 @@
+
+All Appwrite SDKs will have backward compatibility with the Appwrite APIs. In case a new version of the API or product has been released, you should expect your applications to continue working properly without any action from your side. Once we release a major version of the SDK and you decide to upgrade, do look in the changelog of the relevant SDK to understand what changes have been made and what adjustments are required.
+
+We provide early notice to developers and slowly introduce breaking changes to let developers adjust their application at a reasonable pace. 
+We will also continue to support the last five major versions of each SDK to provide developers with more flexibility and time to adjust their apps to take advantage of new features.
+
+# Self-hosted versioning {% #self-hosted-versioning %}
+
+When you self-host Appwrite, we also follow the Semantic Versioning (semver) protocol for versioning our releases. This means that we assign version numbers using a three-part system: major, minor, and patch. The major version changes when we make significant changes to our API or product, which may require significant changes to developers' apps. The minor version changes when we add new features or functionality that do not significantly impact developers' existing apps. Finally, the patch version changes when we make bug fixes or minor improvements.
+
+All the self-hosted versions of Appwrite `>=1.x.x` continue to have support and backward compatibility with the Appwrite API and SDKs within each major version. 
+In case a new version of the product has been released and you decide to update, you should expect your applications to continue working properly without any action from your side.
+
+Once we release a version of the product and you decide to upgrade, look in the changelog to understand if your version requires migration of data from your previous setup. 
+This is usually required when we make adjustments to the under the hood data structure for supporting new features and improving maintainability. 
+If this is the case, you could use our built-in migration tool for helping you to upgrade your self-hosted Appwrite version.
+
+{% arrow_link href="/docs/advanced/self-hosting/update" %}
+Learn more about updating self-hosted Appwrite
+{% /arrow_link %}
+
+# Runtime versioning {% #runtime-versioning %}
+
+Appwrite runtimes are built around a combination of operating system, programming language, and software libraries that are subject to maintenance and security updates.
+Appwrite will support and maintain runtimes as a package, which covers the specific combination of operating system, programming language, and software libraries.
+
+Appwrite will support the latest stable versions of our serverless runtime environments for a minimum of 24 months after its initial release as long as security updates for components of the specific runtime are still provided. 
+You can review the list of supported runtimes on Appwrite.
```

Comment:
link to https://appwrite.io/docs/products/functions/runtimes

Reply:
Done